README.md

```diff
@@ -235,6 +235,34 @@ Something like this:
 copy_to_clipboard "some_id" # some_id is HTML id of element. Content of this element will be copied to the clipboard after pressing the button
 ```
 
+# Development
+
+To start developing you should:
+
+#### 1. install gems
+
+```shell
+bundle install
+```
+
+Also need to install [ImageMagick](https://imagemagick.org) for [rmagick](https://github.com/rmagick/rmagick). You can read the manual for your OS [here](https://github.com/rmagick/rmagick#prerequisites)
```

Comment:
Some misspells here:

* `Also, you need to...`.
* Let's replace `You can read the manual` with `Read the manual`